src/components/organization/OrganizationPartner.js

```diff
@@ -0,0 +1,29 @@
+import {Box, Card, CardContent, Divider, Typography} from "@material-ui/core";
+import {colors} from "../../utils/colors";
+
+export const OrganizationPartner = ({firstname, lastname, mail}) => {
```

Comment:
Wiem, że to js, ale tu też obowiązują zasady XD `firstName` i `lastName`, a nie `firstname` i `lastname`